applications/common/frontend/src/atomic-design-system/00-token/t.size/t.size.scss

```diff
@@ -0,0 +1,27 @@
+/*!
+ * <!--
+ *     Copyright (C) 2024 Dynamic Solutions
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ * -->
+ */
+
+@use '../t.font' as font;
+
+@mixin extra-small {
+  @include font.t-font-size('14')
+}
+
+@mixin extra-small-2 {
+  @include font.t-font-size('12')
+}
```

Comment:
I've been asked to do so, but I'm not a frontend expert enough to review this PR with full condifence 💭